python/holders/src/main.py

```diff
@@ -0,0 +1,13 @@
+# PNG ADDRESS
```

Comment:
We should get these constants from the SDK repository. Else we're duplicating code

Reply:
I haven't ported this code to multichain yet and in python I don't have access to npm/yarn packages

Reply:
You don't need access to npm/yarn. You can just stream this URL https://raw.githubusercontent.com/pangolindex/sdk/master/src/chains.ts into local memory and then parse it to extract the details